demos/demo05_reuse_assemble.py

```diff
@@ -51,14 +51,14 @@ def rhs_form2(m,p):
 PDE1 = cuqipy_fenics.pde.SteadyStateLinearFEniCSPDE( (lhs_form, rhs_form1), mesh, 
         parameter_function_space=parameter_function_space,
         solution_function_space=solution_function_space,
-        dirichlet_bc=dirichlet_bc,
+        dirichlet_bcs=dirichlet_bc,
```

Comment:
In relation to the discussion above of the naming "bcs", it appears a bit confusing/inconsistent that one here is plural "bcs" and the other "bc".